src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  width: 100%;
 }
+
+* {
+  --color-line: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 10px;
+  list-style: none;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  padding: 10px;
+}
+
+.nav__link:hover {
+  text-decoration: underline var(--color-line);
+}
+
+.is-active {
+  color: var(--color-line);
+}
+
+.line {
+  position: relative;
+}
+
+.line::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  background-color: var(--color-line);
+  border: 1px solid;
+  border-radius: 20px;
+}
+/* @media (max-width: 1024px){
```

Comment:
Media queries for 1024px and 1200px are empty and commented out. Adaptive styles for these screen sizes should be implemented as per the task description.